src/main/java/org/programmers/dev/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,23 @@
+package org.programmers.dev.exception;
+
+import org.programmers.dev.exception.dto.ErrorResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class GlobalExceptionHandler {
+
+    @ExceptionHandler({
+        PostValidationException.class,
+        PostNotFoundException.class
+    })
+    public ResponseEntity<ErrorResponse> handle400Exception(Exception e) {
```

Comment:
```suggestion
    public ResponseEntity<ErrorResponse> handle400Exception(RuntimeException e) {
```

ExceptionHandler에 선언된 예외들이 RuntimeException 예외를 상속받아 사용되어 인자 값으로 RuntimeException을 사용하는 것이 좋겠네요.!